DB/db.sql

```diff
@@ -8,7 +8,8 @@ CREATE TABLE utente (
        username VARCHAR(30) PRIMARY KEY,
        password VARCHAR(30) NOT NULL,
        email VARCHAR(30) NOT NULL,
-       isAdmin BOOLEAN
+       isAdmin BOOLEAN,
+       saldo FLOAT NOT NULL
 );
```

Comment:
Il tipo di saldo dovrebbe essere DECIMAL(10,2), il float non è adatto alle currency.

Reply:
sono d'accordo, solo una cosa, il 2 indica il numero di cifre dopo la virgola giusto? perchè se fosse cosi, non dovrebbe essere tipo DECIMAL(10,10) dato che l'utente paga con una criptovaluta (esempio: 9,04045 ETH) e non in $/€? @Giuseppe5000

Reply:
Si ci sta